### DISCUSSION TITLE
Persisted state

### DISCUSSION THREAD

**Discussion by andreyddk** [DioxusLabs/dioxus#843] [Q&A]
Hi! I trying to find solution for persisted state like local-storage or something else.

Its posable to implement?

**Comment by ealmloff** [comment]
You can use [gloo storage](https://docs.rs/gloo-storage/latest/gloo_storage/index.html) to interact with the local storage. If you wrap it in a hook with read and write methods you can automatically sync with local storage

(I am also working on a more ergonomic solution for storage in Dioxus [here](https://github.com/Demonthos/dioxus-storage). The [use_persistant](https://github.com/Demonthos/dioxus-storage/blob/master/examples/persistant.rs) hook automatically syncs with local storage, but it is not at all stable right now)
